src/picknik_ur_site_config/objectives/open_cabinet_door_affordance.xml

```diff
@@ -3,13 +3,17 @@
     <BehaviorTree ID="Open Cabinet Door" _description="Open a cabinet door by grasping and pulling its handle">
         <Control ID="Sequence" name="Setup">
             <!-- Tare the force torque sensor -->
+            <!-- Wrap in a ForceSuccess as Sim doesn't have the zero_ftsensor service -->
+            <!-- See https://github.com/PickNikRobotics/moveit_studio/issues/3406 -->
             <Decorator ID="ForceSuccess">
                 <Action ID="CallTriggerService" service_name="/io_and_status_controller/zero_ftsensor" />
             </Decorator>
             <!-- Set admittance control parameters at the start of the Objective -->
-            <Action ID="UpdateAdmittanceController" config_file_name="open_door_admittance_controller_config.yaml"/>
+            <Decorator ID="ForceSuccess">
```

Comment:
Maybe I pointed at the wrong decorator, but I meant to add a comment to this one as well. This one needs to be force success'd because sim doesn't have admittance controller loaded.

Reply:
Gotcha, updated others to follow suit.